uarray/backend.py

```diff
@@ -174,12 +174,25 @@ def replace_dispatchables(
         args, kwargs = argument_replacer(args, kwargs, tuple(replaced_args))
 
         kwargs = {k: kwargs[k] for k in opts if k in kwargs}
```

Comment:
It might be nice to factor this out into the same function as `*args`.

Reply:
This step is done after argument replacement whereas I have put my step before `args` reaches the backend. This way it's only done once.